takanon_extractions/dynamic_extractions/extract_sources/ethic_rules/החלטת ועדת הכנסת בדבר כללי אתיקה לחברי הכנסת_structure_content.json

```diff
@@ -0,0 +1,336 @@
+{
+  "metadata": {
+    "input_file": "extract_sources/ethic_rules/החלטת ועדת הכנסת בדבר כללי אתיקה לחברי הכנסת.html",
+    "document_name": "החלטת ועדת הכנסת בדבר כללי אתיקה לחברי הכנסת",
+    "environment": "staging",
+    "model": "gpt-4.1",
+    "max_tokens": 32000,
+    "total_items": 45,
+    "structure_type": "nested_hierarchy",
+    "mark_type": "סעיף"
+  },
+  "structure": [
+    {
+      "depth": 1,
+      "section_name": "תוכן עניינים",
+      "html_id": null,
+      "section_type": "תוכן עניינים"
+    },
+    {
+      "depth": 1,
+      "section_name": "פרק א׳: פרשנות",
+      "html_id": null,
+      "section_type": "פרק",
+      "children": [
+        {
+          "depth": 2,
+          "section_name": "סעיף 1",
+          "html_id": "סעיף_1",
+          "section_type": "סעיף",
+          "content": "[1.](https://he.wikisource.org/wiki/%D7%9B%D7%9C%D7%9C%D7%99_%D7%90%D7%AA%D7%99%D7%A7%D7%94_%D7%9C%D7%97%D7%91%D7%A8%D7%99_%D7%94%D7%9B%D7%A0%D7%A1%D7%AA#%D7%A1%D7%A2%D7%99%D7%A3_1)\n\nהגדרות [תיקון: תשע״ב]\n\nבכללים אלה:\n\n”בן משפחה“ – בן־זוג, ילד קטין או כל קרוב שפרנסתו על חבר הכנסת;\n\n”ועדת האתיקה“ – ועדת האתיקה שמונתה לפי [סעיף 18](https://he.wikisource.org/wiki/%D7%9B%D7%9C%D7%9C%D7%99_%D7%90%D7%AA%D7%99%D7%A7%D7%94_%D7%9C%D7%97%D7%91%D7%A8%D7%99_%D7%94%D7%9B%D7%A0%D7%A1%D7%AA#%D7%A1%D7%A2%D7%99%D7%A3_18);\n\n”חוק החסינות“ – [חוק חסינות חברי הכנסת, זכויותיהם וחובותיהם, התשי״א–1951](https://he.wikisource.org/wiki/%D7%97%D7%95%D7%A7_%D7%97%D7%A1%D7%99%D7%A0%D7%95%D7%AA_%D7%97%D7%91%D7%A8%D7%99_%D7%94%D7%9B%D7%A0%D7%A1%D7%AA,_%D7%96%D7%9B%D7%95%D7%99%D7%95%D7%AA%D7%99%D7%94%D7%9D_%D7%95%D7%97%D7%95%D7%91%D7%95%D7%AA%D7%99%D7%94%D7%9D \"חוק חסינות חברי הכנסת, זכויותיהם וחובותיהם\");\n\n”טובת הנאה“ – לרבות כסף, שווה כסף, נכס, מוצר או שירות;\n\n”לקוח“ – מי שנותן, או אמור לתת, שכר או תמורה בעד שירותים;\n\n”ענין אישי“ – הנאה חמרית, ישירה או עקיפה, של חבר הכנסת, של קרובו, של לקוחו, או של חבר־בני־אדם, מואגד ושאינו מואגד, העוסק בפעילות המכוונת להשגת רווחים שחבר הכנסת חבר בהנהלתו או נושא משרה בו, למעט ענין הנוגע לענף משקי שחבר הכנסת משתייך אליו;\n\n”קרוב“ – בן־זוג, הורה, צאצא וכל קרוב משפחה שפרנסתו על חבר הכנסת;\n\n”רשות ממשל“ – המדינה, רשות מקומית, תאגיד שהוקם מכוח דין, למעט בית משפט ובית־דין;\n\n”רשות ציבורית“ – משרד ממשלתי, רשות מקומית, תאגיד שהוקם לפי חוק, חברה ממשלתית או חברה עירונית וכן חברת־בת של חברה כאמור;\n\n”תמורה“ – תמורה כספית או תמורה חומרית אחרת.\n\n## [פרק ב׳: כללי יסוד](https://he.wikisource.org/wiki/%D7%9B%D7%9C%D7%9C%D7%99_%D7%90%D7%AA%D7%99%D7%A7%D7%94_%D7%9C%D7%97%D7%91%D7%A8%D7%99_%D7%94%D7%9B%D7%A0%D7%A1%D7%AA#%D7%A4%D7%A8%D7%A7_%D7%91)"
```

Comment:
Please ensure the links are not using the 'percent encoding' but appear as hebrew.

Reply:
I reviewed the recent outputs (_structure.json, _structure_content.json, and pipeline_metadata.json). All links and file paths now appear in Hebrew, without percent encoding.  It looks to me like this issue has been resolved with the latest fixes.

Reply:
See the next PR - it still looks like that there...